src/Event/Listeners/OracleSessionInit.php

```diff
@@ -38,8 +38,13 @@ class OracleSessionInit implements EventSubscriber
     ];
 
     /** @param string[] $oracleSessionVars */
-    public function __construct(array $oracleSessionVars = [])
+    public function __construct(array $oracleSessionVars = [], bool $useHighPrecisionTimestampFormatString = false)
```

Comment:
I don't think we need to patch this deprecated event listener. Application that want to opt into the more precise timestamp format should simply switch to the middleware.

Reply:
Opt-in for oracle and db2 now refactored. I've kept the event listener patch as there's no need to touch the constructor now. Happy to revert this bit of the oracle patch if necessary.